Supports lsphp82

DuCun opened the pull request.

Code-Egg commented:
Thanks for the PR, due to some minor LSPHP82 package issues that need to be confirmed and fixed. Could you keep LSPHPVER=81 at the moment? 

https://github.com/litespeedtech/ols1clk/pull/59/commits/443524cf223953e2cb75a64d151696f181857d30#diff-f01f45bef6c1cf6e78a7140caeb16a27efa624ebc1bfaf09b6b7776468e4e448L65-R65

DuCun commented:
> Thanks for the PR, due to some minor LSPHP82 package issues that need to be confirmed and fixed. Could you keep LSPHPVER=81 at the moment?
> 
> [443524c#diff-f01f45bef6c1cf6e78a7140caeb16a27efa624ebc1bfaf09b6b7776468e4e448L65-R65](https://github.com/litespeedtech/ols1clk/commit/443524cf223953e2cb75a64d151696f181857d30#diff-f01f45bef6c1cf6e78a7140caeb16a27efa624ebc1bfaf09b6b7776468e4e448L65-R65)

Oh! I forgot that some of the packages don't support php8.2 yet and I have changed them again.
